packages/toolkit/src/query/index.ts

```diff
@@ -26,5 +29,6 @@ export type { CreateApi, CreateApiOptions } from './createApi'
 export { buildCreateApi } from './createApi'
 export { fakeBaseQuery } from './fakeBaseQuery'
 export { copyWithStructuralSharing } from './utils/copyWithStructuralSharing'
-export { createApi, coreModule } from './core'
+export { createApi, coreModule, coreModuleName } from './core'
 export { defaultSerializeQueryArgs } from './defaultSerializeQueryArgs'
+export * from './tsHelpers';
```

Comment:
Please try to keep it to a minimum and check what is actually needed. From what I gather, that might actually only be `symbol` types. 
We don't want to export too much, since at that point implementation details become a public API that we have to try to keep stable and keep supporting pretty much indefinitely.

Reply:
Makes sense; you got it! I'll pin point it.

Maybe you're right about the symbols and only `coreModuleName` needs to be exposed. TypeScript has a hard time making it clear.

Here's a very simple reproduction of all the **portability errors** I was seeing in my project:
https://codesandbox.io/p/sandbox/rtk-type-portability-j99lyh?file=%2Fsrc%2Frtk%2FmyApiBaseQuery.ts%3A11%2C3

^ Additional comments in the sandbox to see what might've lead me astray.